Gemfile

```diff
@@ -126,3 +126,5 @@ gem 'csl-styles'
 gem 'acts-as-taggable-on'
 gem 'mods_display'
 gem 'slack-ruby-client'
+gem 'blacklight-oembed', '~> 0.2'
+gem 'rack-test', '0.7.0'
```

Comment:
can you add a comment here on why we're pinning to a specific version? so we can unpin with the issue is resolved down the line...

Reply:
Sure thing. I added it in the commit message, but probably should have put it here too. 

Added. Thanks for catching.